src/parser/ParseException.h

```diff
@@ -22,8 +22,8 @@ struct ExceptionMetadata {
   // Start and Stop Index of the clause that cause the exception in query_.
   size_t startIndex_;
   size_t stopIndex_;
-  // Start of the clause expressed as line and position in the line relative to
-  // query_.
+  // Start of the clause expressed as line and position in the line (0-based)
```

Comment:
Thanks for clarifying this. I think the lines are 1-based, either way, also integrate the comment there. 
And maybe add a commment above, that the character at position `stopIndex` is part of the clause
and fix the typo `Start and stop index....` (capitalization errors).